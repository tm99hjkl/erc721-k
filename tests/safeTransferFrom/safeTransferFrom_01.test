makeAccount(1)
makeAccount(2)
makeNFT(999, 1)
makeNFT(99, 1)
safeTransferFrom(1,2,999)