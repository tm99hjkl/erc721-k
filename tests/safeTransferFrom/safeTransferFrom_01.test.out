<ERC721>
  <k>
    safeTransferFrom ( 1 , 2 , 999 ) ~> .
  </k>
  <caller>
    0
  </caller>
  <accounts>
    <account>
      <accountID>
        1
      </accountID>
      <balance>
        2
      </balance>
      <operate>
        1 |-> true
      </operate>
    </account> <account>
      <accountID>
        2
      </accountID>
      <balance>
        0
      </balance>
      <operate>
        2 |-> true
      </operate>
    </account>
  </accounts>
  <nfts>
    <nft>
      <nftID>
        99
      </nftID>
      <nftOwner>
        1
      </nftOwner>
      <nftController>
        1
      </nftController>
    </nft> <nft>
      <nftID>
        999
      </nftID>
      <nftOwner>
        1
      </nftOwner>
      <nftController>
        1
      </nftController>
    </nft>
  </nfts>
  <log>
    Event: 
  </log>
</ERC721>
